api-test/helpers/circles.ts

```diff
@@ -10,6 +10,8 @@ const createCircleSchemaInput = z
   .object({
     name: z.string().min(3).max(255),
     organization_id: z.number().int().positive(),
+    telegram_id: z.string().optional(),
```

Comment:
@sshmm do these need to be part of this PR?

Reply:
yeah, they're for the test